dev/domain-controller/FinalizeDeployment.ps1

```diff
@@ -1,10 +1,21 @@
 ﻿Param(
+  [string]$rgName,
   [string]$domainUserUPN,
   [string]$domainPassword,
   [string]$machineToJoin,
-  [string]$groupToJoin
+  [string]$groupToJoin,
+  [string]$azureUserName, 
+  [string]$azurePassword, 
```

Comment:
Oh right- I'd make the whole 'commandToExecute' a protected setting. Duh. We're okay. Thanks for the reminder.